cli/cdsctl/admin_broadcasts.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ovh/cds/cli"
+	"github.com/ovh/cds/sdk"
+)
+
+var (
+	adminBroadcastsCmd = cli.Command{
+		Name:  "broadcasts",
+		Short: "Manage CDS broadcasts",
+	}
+
+	adminBroadcasts = cli.NewCommand(adminBroadcastsCmd, nil,
+		[]*cobra.Command{
+			cli.NewListCommand(adminBroadcastListCmd, adminBroadcastListRun, nil),
+			cli.NewGetCommand(adminBroadcastShowCmd, adminBroadcastShowRun, nil),
+			cli.NewCommand(adminBroadcastDeleteCmd, adminBroadcastDeleteRun, nil),
+			cli.NewCommand(adminBroadcastCreateCmd, adminBroadcastCreateRun, nil),
+		})
+)
+
+var adminBroadcastCreateCmd = cli.Command{
+	Name:  "create",
+	Short: "Create a CDS broadcast",
+	Args: []cli.Arg{
+		{Name: "level"},
+		{Name: "title"},
+		{Name: "content"},
```

Comment:
i could be nice to be able to user sdtin for the content
like `cdsctl admin broadcast create info "Title" < changelog.md`